scripts/publish-version.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env sh
```

Comment:
❤️

Reply:
POSIX :heart: 

You reminded me that I forgot to pass it through [ShellCheck](https://www.shellcheck.net/): [it’s done now](https://github.com/aragon/connect/pull/188/commits/0419390b34fde56351e0f04782204e50d9cbfae6) :smile: